File: AddressBook/AddressBook/Dtos/ChatLabel.cs
using AddressBook.Common.Paging;
using System.ComponentModel.DataAnnotations;

namespace AddressBook.Dtos
{
    public class ChatLabelRes
    {
        public int ChatLabelId { get; set; }
        public string? Label { get; set; }
    }

    public class ChatLabelReqEdit
    {
        [Required, MaxLength(20)]
        public string? Label { get; set; }
    }

    public class ChatLabelReqSearch : PagedReq
    {

    }
}

File: AddressBook/react-client/src/dtos/ChatLabel.ts
import Common from "../utility/Common";
import { PagedReq } from "./PagedReq";

export interface ChatLabelRes {
  chatLabelId?: number;
  label?: string;
}

export class ChatLabelReqEdit {
  label?: string = "";
}

export class ChatLabelReqSearch extends PagedReq {
  constructor(
    {
      pageNumber = 1,
      pageSize = Common.DEFAULT_PAGE_SIZE,
      orderBy,
      searchText = "",
    }: PagedReq,
    {}
  ) {
    super({
      pageNumber: pageNumber,
      pageSize: pageSize,
      orderBy: orderBy,
      searchText: searchText,
    });
  }
}